build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
okay there are still many uses for gradle like dependency management and structuring modules and many more.It is basically a jvm based build tool.